apps/survey/contracts/Survey.sol

```diff
@@ -0,0 +1,250 @@
+pragma solidity 0.4.18;
+
+import "@aragon/os/contracts/apps/AragonApp.sol";
+
+import "@aragon/os/contracts/lib/minime/MiniMeToken.sol";
+import "@aragon/os/contracts/lib/zeppelin/math/SafeMath.sol";
+import "@aragon/os/contracts/lib/zeppelin/math/SafeMath64.sol";
+import "@aragon/os/contracts/lib/misc/Migrations.sol";
+
+
+contract Survey is AragonApp {
+    using SafeMath for uint256;
+    using SafeMath64 for uint64;
+
+    uint256 constant public PCT_BASE = 10 ** 18; // 0% = 0; 1% = 10^16; 100% = 10^18
+    uint256 constant public NO_VOTE = 0;
+    bytes32 constant public CREATE_SURVEYS_ROLE = keccak256("CREATE_SURVEYS_ROLE");
+    bytes32 constant public MODIFY_PARTICIPATION_ROLE = keccak256("MODIFY_PARTICIPATION_ROLE");
+
+    MiniMeToken public token;
+    uint256 public minAcceptParticipationPct;
+    uint64 public surveyTime;
+
+    struct OptionCast {
+        uint256 optionId;
+        uint256 stake;
+    }
+
+    /** To allow multiple option votes.
+     * Index 0 will always be for NO_VOTE option
+     * option Ids must be in ascending order
+     */
```

Comment:
Yes, I was using /* ... */ but @izqui asked me to change it. See his comment here:
https://github.com/aragon/aragon-apps/pull/287#discussion_r185331000
As I have said before, we should have a clear guide of our style, especially when it differs from "official" Solidity one (those comments are perfectly valid according to it: http://solidity.readthedocs.io/en/v0.4.23/layout-of-source-files.html#comments) and enforced with `lint` as much as possible. I feel these kind of discussions distracting from the important things and sometimes even a little bit frustrating.
I personally like `/* ... */` for multi line and `//` for one line, but as I have said before too, I will be glad to adapt to any agreed style if it's clearly stated somewhere.

Reply:
Totally agree, although I don't think the linters are super smart about this right now.

Let's use `/*` or `//` for multi line blocks, but definitely not `///` or `/**` since those are **explicitly** for natspec.